com.unity.render-pipelines.universal/Editor/ShaderGraph/Includes/DepthNormalsOnlyPass.hlsl

```diff
@@ -19,7 +19,7 @@ half4 frag(PackedVaryings packedInput) : SV_TARGET
     SurfaceDescriptionInputs surfaceDescriptionInputs = BuildSurfaceDescriptionInputs(unpacked);
     SurfaceDescription surfaceDescription = SurfaceDescriptionFunction(surfaceDescriptionInputs);
 
-    #if _AlphaClip
+    #if _ALPHATEST_ON
```

Comment:
Was this a bug?

Reply:
_AlphaClip was the name of the #define that the old URP Target used to control alpha clipping.
Since this is upgraded to a (potential) keyword now, we rename it to match the name used by URP Lit.shader.

Reply:
This doesn't break shadergraphs because it was always a generated #define, not exposed to user in any way.